src/antsibull_changelog/rst.py

```diff
@@ -15,12 +15,14 @@ class RstBuilder:
     """
     Simple reStructuredText (RST) builder.
     """
+    lines: List[str]
```

Comment:
I suppose this will be changed to `list[str]` in the follow up typing PR.

Reply:
Yes (see #94). This PR only has minimal changes to make CI pass.